run_docker.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+# Here you can change TAG for Docker image 
+IMAGE_TAG="latest"
+
+function usage() {
+    echo "Run Docker Container for building Yocto"
+    echo ""
+    echo "Command line arguments:"
+    echo -e "\t--workspace, -w\n\t\tWorkspace"
+    echo -e "\t--suffix, -s\n\t\tSuffix for Container name (optional)"
+    echo -e "\t--docker_image, -d\n\t\tDocker image that should be used for building Yocto"
+    echo ""
+
+    exit 2
+}
+
+options=$(getopt -o hw:s:d: \
+    --long workspace:,suffix:,docker_image: \
+    --long help \
+    -- "$@")
+
+# Map option variable to positional arguments (i.e. $1, $2 ...)
+eval set -- "$options"
+
+while :; do
+    case "$1" in
+    -h | --help) usage ;;
+    -w | --workspace)
+        WORKSPACE=$(readlink -f "$2")
+        shift 2
+        ;;
+    -s | --suffix)
+        SUFFIX="$2"
+        shift 2
+        ;;
+    -d | --docker_image)
+        DOCKER_IMAGE_NAME="$2"
+        shift 2
+        ;;
+    # -- means the end of the arguments; drop this, and break out of the while loop
+    --)
+        shift
+        break
+        ;;
+    # If invalid options were passed, then getopt should have reported an error,
+    # which we checked as VALID_ARGUMENTS when getopt was called...
+    *)
+        echo "Unexpected option: $1 - this should not happen."
```

Comment:
But it happens, so display the proper error here.

Reply:
after this message `usage` shows what exactly args should be used.
to do not make it unclear - removed part of message ` - this should not happen.`